chains/ngmi/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+blockExplorers:
+  - apiUrl: https://ngmi-2712765173321000-1.sagaexplorer.io/api-docs
+    family: blockscout
+    name: NGMI Explorer
+    url: https://ngmi-2712765173321000-1.sagaexplorer.io
+blocks:
+  confirmations: 1
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 2712765173321000
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: NGMI
+domainId: 2712765173321000
+gasCurrencyCoinGeckoId: ngmi
```

Comment:
double checking this is the right gecko entry? https://www.coingecko.com/en/coins/ngmi

Reply:
that doesnt look right :/ added you to the dev chat (Land Labor Capitol)

Reply:
no coingeckoid yet